kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -160,18 +161,28 @@ spec:
   # If the domain home is in the image
   domainHomeInImage: true
   # The Operator currently does not support other images
-  image: "domain-home-in-image:latest"
-  # imagePullPolicy defaults to "Never"
-  imagePullPolicy: "Never"
+  image: "domain-home-in-image"
+  # imagePullPolicy defaults to "Always" if image version is :latest
+  imagePullPolicy: "IfNotPresent"
+  # Identify which Secret contains the credentials for pulling an image
+  #imagePullSecrets:
+  #- name:
   # Identify which Secret contains the WebLogic Admin credentials (note that there is an example of
   # how to create that Secret at the end of this file)
   webLogicCredentialsSecret:
     name: domain1-weblogic-credentials
   # Whether to include the server out file into the pod's stdout, default is true
   includeServerOutInPodLog: true
+  # Whether to enable log home
```

Comment:
"Whether to enable log home.  Defaults to false if domainHomeInImage is true, and to true when domainHomeInImage is false."

Reply:
This is from the sample's domain-template.yaml.  I prefer not making the sample template to play the role of domain schema documentation.  Tom and I chatted about this over the phone. The decision is to keep the template sample comments simple. 
Made a change base don this.